README.md

```diff
@@ -1,12 +1,12 @@
 # This is the https://regolith-desktop.com website
 
-This site uses the [Hugo static site generator](https://gohugo.io/) and the [Book theme](https://themes.gohugo.io/themes/hugo-book/).
+This site uses the [Hugo static site generator](https://gohugo.io/) and the [Hextra theme](https://themes.gohugo.io/themes/hextra/).
 
 ## Local Development
 
 Please ensure you have the following installed:
 
-- Hugo **extended** **>= 0.80.0** (otherwise PostCSS won't work)
+- Hugo **extended** **>= 0.140.0** (otherwise PostCSS won't work)
```

Comment:
I notice I'm running an older version of Hugo on `noble`.  I guess I'll need to build the tool from source going forward.  

```
hugo/noble-updates,noble-security,now 0.123.7-1ubuntu0.2 amd64 [installed]
```

Reply:
Or you can simply use `docker compose up`